ngraph/core/include/ngraph/op/unsqueeze.hpp

```diff
@@ -52,8 +47,5 @@ namespace ngraph
                     clone_with_new_inputs(const OutputVector& new_args) const override;
             };
         }
-        using v0::Unsqueeze;
```

Comment:
I have thought more about this using. And it is not correct that we remove it.

If we remove it we break the backward compatibility. It is why I recommend to revert this change. Sorry for inconvinience.

We can remove `using` only for deprecated operations.

Reply:
Fixed.